blocks/footer/footer.css

```diff
@@ -78,18 +98,18 @@ footer .columns ul li:last-of-type {
 
 footer .columns ul li a[href] {
   text-decoration: none;
-  color: var(--mt-global-color-accent-quinary);
+  color: var(--mt-background-color-primary);
```

Comment:
Header color should be `--mt-heading-color-inverted` instead of `--mt-background-color-primary`